### PR TITLE
Support .yaml config files and the routes.yaml file

### LINKED ISSUE #1198: Support the new Symfony config files
As discussed in Mumble on January 16th, we want to support the new Symfony configuration files.

* `routing.yml` -> `routes.yaml`
* `config.yml` -> `config.yaml`
* `services.yml` -> `services.yaml`
* and so on

### PR DESCRIPTION
**Pull request by aschempp**
fixes #1198

### CODE REVIEW

**Review by bytehead — DISMISSED** [review]

**Review by Toflar — DISMISSED** [review]

**Review by leofeyer — DISMISSED** [review]

**Review by bytehead — DISMISSED** [review]

File: manager-bundle/src/Routing/RouteLoader.php [hunk 1]
```diff
@@ -89,17 +89,25 @@ function (RouteCollection $collection, RoutingPluginInterface $plugin): RouteCol
...
+                }
+
+                return $this->rootDir.'/config/'.$file;
+            }
         }
 
         // Fallback to the legacy config file (see #566)
-        if (file_exists($this->rootDir.'/app/config/'.$file)) {
-            @trigger_error(sprintf('Storing the "%s" file in the "app/config" folder has been deprecated and will no longer work in Contao 5.0. Move it to the "config" folder instead.', $file), E_USER_DEPRECATED);
+        foreach (['routing.yaml', 'routing.yml'] as $file) {
```
Comment (aschempp) [inline manager-bundle/src/Routing/RouteLoader.php]: `routes.y(a)ml` should be supported in the old place as well

Comment (leofeyer) [reply manager-bundle/src/Routing/RouteLoader.php]: Really? I have just asked in the Mumble call and @ausi and @Toflar share my opinion that the new Symfony 4 file does not have to be supported in the old Symfony 3 config location.

Comment (aschempp) [reply manager-bundle/src/Routing/RouteLoader.php]: I only remember we said „keep it consistent everywhere“? 🤷‍♂️

Comment (m-vo) [reply manager-bundle/src/Routing/RouteLoader.php]: Both true :-)

**Review by m-vo — DISMISSED** [review]

File: manager-bundle/src/Routing/RouteLoader.php (hunk 1 above)
Comment (leofeyer) [reply manager-bundle/src/Routing/RouteLoader.php]: Yes, keep it consistent as in "allow .yml and .yaml" everywhere. But not "allow Symfony 4 files in the deprecated Symfony 3 file location".

Comment (aschempp) [reply manager-bundle/src/Routing/RouteLoader.php]: /cc @contao/developers

Comment (ausi) [reply manager-bundle/src/Routing/RouteLoader.php]: I agree with @leofeyer

Comment (leofeyer) [reply manager-bundle/src/Routing/RouteLoader.php]: Auf mehrfachen Wunsch eines einzelnen: a1a5332c549a6f108ae3f2004e25c3cee9c2b16e 🤷‍♂

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Pull request merged**